ess/src/API/EMBC.ESS/Resources/Supports/SupportRepository.cs

```diff
@@ -128,19 +165,92 @@ private async Task<PotentialDuplicateSupportsQueryResult> Handle(PotentialDuplic
+                                        string.Equals(member.era_lastname?.Trim(), supportMember.era_lastname?.Trim(), StringComparison.OrdinalIgnoreCase))
+                                        scenario = ConflictMessageScenario.ExactMatchOnDifferentEssFile;
+                                }
+                                Guid guid = Guid.NewGuid();
+                                era_supportconflictmessage eraConflictMessage = new era_supportconflictmessage
+                                {
+                                    era_supportconflictmessageid = guid,
+                                    era_name = guid.ToString(),
+                                    era_EvacueeSupport = support,
+                                    era_ESSFile = file,
+                                    era_ESSTask = file.era_TaskId,
+                                    era_Registrant = file.era_Registrant,
+
+                                    // era_Registrant = support.era_EvacuationFileId.era_Registrant,
+                                    era_Responder = responder,
+
+                                    era_evacueedob = member.era_dateofbirth,
+                                    era_evacueename = $"{member.era_firstname} {member.era_lastname}",
+                                    createdon = DateTime.Now,
+
+                                    era_MatchedESSFile = support.era_EvacuationFileId,
+                                    era_matchedname = $"{supportMember.era_firstname} {supportMember.era_lastname}",
+                                    era_matcheddob = supportMember.era_dateofbirth,
+                                    era_scenario = (int)scenario,
+                                };
+
+                                //CreateConflictMessage(ctx, eraConflictMessage, ct);
+
+                                var supportTypeEnum = (SupportType)support.era_supporttype.Value;
+                                var supportTypeDisplayName = supportTypeEnum.GetType()
+                                    .GetField(supportTypeEnum.ToString())
+                                    ?.GetCustomAttributes(typeof(DisplayAttribute), false)
+                                    .Cast<DisplayAttribute>()
+                                    .FirstOrDefault()?.Name;
+
+                                if (string.IsNullOrEmpty(supportTypeDisplayName))
+                                {
+                                    supportTypeDisplayName = "None";
+                                }
+
+                                potentianExtendedDuplicateSupports.Add(
+                                new DuplicateSupportResult
+                                {
+                                    duplicateSupportScenario = (int)scenario,
+                                    essFileId = tempSupport.era_EvacuationFileId.era_name,
+                                    supportStartDate = tempSupport.era_validfrom.Value.UtcDateTime.ToPST().ToString("MM-dd-yyyy"),
```

Comment:
Why not instead of modifying the time in backend, we do it in the front end? I know  other parts of the code are doing the same. Other parts are using DateTime.UtcNow, and others just DateTime. We should have all dates in the same format with just DateTime and handle changes in FrontEnd.